### PR TITLE
fix: reset default volume when the equalizer disabled

### PR DESCRIPTION
**Pull request by kt286**
reset default volume when the equalizer disabled

Log: reset default volume when the equalizer disabled

### CODE REVIEW

**Review by wyu71 — COMMENTED** [review]
The default index of the equalizer is 1, and the index 0 is custom

File: src/music-player/core/player.cpp [hunk 1]
```diff
@@ -999,12 +999,19 @@ void Player::setEqualizerCurMode(int curIndex)
 {
     //非自定义模式时
     if (curIndex != 0) {
```
[the remark this answers is not in the document]
Comment (kt286) [reply src/music-player/core/player.cpp:1001]: 万一后边 = 0 的时候有其他判断呢？
我再改一下，>0  <0  后边也可以再加 =0

File: src/music-player/core/player.cpp [hunk 2]
```diff
@@ -998,14 +998,19 @@ void Player::setEqualizerbauds(int index, int val)
...
         //设置放大值
         m_basePlayer->setPreamplification(m_basePlayer->preamplification());
         for (int i = 0 ; i < 10; i++) {
             //设置频率值
             m_basePlayer->setAmplificationForBandAt(m_basePlayer->amplificationForBandAt(uint(i)), uint(i));
         }
+    } else if (curIndex == -1) { // 关闭均衡器时
+        for (int i = 0 ; i < 10; i++) {
+            //设置频率值
+            m_basePlayer->setAmplificationForBandAt(uint(10), uint(i));
```
Comment (wyu71) [inline src/music-player/core/player.cpp:1012]: 10可以改为0,再增加m_basePlayer->setPreamplification(12);

**Review by wyu71 — APPROVED** [review]
/+1

**Pull request merged**